packages/firestore/lib/index.d.ts

```diff
@@ -86,11 +86,18 @@ export namespace FirebaseFirestoreTypes {
     toUint8Array(): Uint8Array;
   }
 
+  /**
+   * A `DocumentData` object represents the data in a document.
+   */
+  export interface DocumentData {
+    [key: string]: DocumentFieldType;
```

Comment:
I integrated the changes into my work project for testing and found them to be breaking
They diverge from the web SDK which is the basis of comparison for the APIs and typing here: https://github.com/firebase/firebase-js-sdk/blob/f506e18facf8f47eff61cf83a98af4b1f2e94319/packages/firestore/exp-types/index.d.ts#L23

```suggestion
    [key: string]: any;
```

When I use this suggestion, it works though, non-breaking, and the types seem to match the upstream 

I will apply this suggestion then this is good to go